doc/man1/openssl-ca.pod.in

```diff
@@ -788,6 +789,8 @@ retained mainly for compatibility reasons.
 
 The B<-section> option was added in OpenSSL 3.0.0.
 
+The B<-certform> option has become obsolete in OpenSSL 3.0.0 and has no effect.
```

Comment:
You should also mention "all B<-keyform> values except B<ENGINE>..."

Reply:
Done here and also for other apps where applicable.